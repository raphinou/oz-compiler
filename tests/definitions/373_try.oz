% try..finally statement

% Exception raised
try {Show a} raise stopHere end {Show b}
catch
   stopHere then
   {Show 'stopped'}
finally
   {Show 'and finished'}
end

% Success
try {Show a} {Show b}
catch
   stopHere then
   {Show 'stopped'}
finally
   {Show 'and finished'}
end
