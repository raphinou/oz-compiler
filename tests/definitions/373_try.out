a
stopped
'and finished'
a
b
'and finished'
